Astral Pathfinder/Models/Planet.cpp

```diff
@@ -0,0 +1,18 @@
+//
+//  Planet.cpp
+//  Astral Pathfinder
+//
+//  Created by Ian Holdeman on 2/10/18.
+//  Copyright © 2018 Ian Holdeman. All rights reserved.
+//
+
+// MARK: Header File
+#include "Planet.hpp"
+
+Planet::Planet() {
+  // Will do stuff eventually
```

Comment:
we can leave this to do nothing if we leave the galaxy class to initialize all the planets